### ISSUE TITLE
floccus is not synchronizing anymore

### ISSUE THREAD

**Issue by taiBsu** [floccusaddon/floccus#1408]
### Which version of floccus are you using?

v4.19.1

### Sync method

Nextcloud Bookmarks

### Which browser are you using? In case you are using the phone App, specify the Android or iOS version and device please.

LibreWolf (Firefox)

### Which version of Nextcloud Bookmarks are you using? (if relevant)

v13.1.0

### Which version of Nextcloud? (if relevant)

v27.0.1

### Describe the Bug

I'm unable to sync my bookmarks anymore, I keep getting errors inside the add-on:

```
 Could not create - #undefined[Customize Firefox](1) parentId: oz5SHVuI8hAC: Invalid bookmark: {"title":"Customize Firefox","type":1,"url":"1","parentGuid":"oz5SHVuI8hAC"} Could not create - #undefined[******](1) parentId: dTTAE_g4tsOR: Invalid bookmark: {"title":"******","type":1,"url":"1","parentGuid":"dTTAE_g4tsOR"} Could not create - #undefined[*******](1) parentId: Axo2nlexErch: Invalid bookmark: {"title":"*****","type":1,"url":"1","parentGuid":"Axo2nlexErch"} Could not create - #undefined[](1) parentId: toolbar_____: Invalid bookmark: {"title":"","type":1,"url":"1","parentGuid":"toolbar_____"} Could not create - #undefined[](1) parentId: toolbar_____: Invalid bookmark: {"title":"","type":1,"url":"1","parentGuid":"toolbar_____"} Could not create - #undefined[](1) parentId: toolbar_____: Invalid bookmark: {"title":"","type":1,"url":"1","parentGuid":"toolbar_____"} Could not create - #undefined[](1) parentId: toolbar_____: Invalid bookmark: {"title":"","type":1,"url":"1","parentGuid":"toolbar_____"} Could not create - #undefined[](1) parentId: toolbar_____: Invalid bookmark: {"title":"","type":1,"url":"1","parentGuid":"toolbar_____"} Could not create - #undefined[](1) parentId: toolbar_____: Invalid bookmark: {"title":"","type":1,"url":"1","parentGuid":"toolbar_____"} Could not create - #undefined[Git](1) parentId: toolbar_____: Invalid bookmark: {"title":"Git","type":1,"url":"1","parentGuid":"toolbar_____"} Could not create - #undefined[](1) parentId: toolbar_____: Invalid bookmark: {"title":"","type":1,"url":"1","parentGuid":"toolbar_____"} Could not create - #undefined[](1) parentId: toolbar_____: Invalid bookmark: {"title":"","type":1,"url":"1","parentGuid":"toolbar_____"} Could not create - #undefined[***](1) parentId: toolbar_____: Invalid bookmark: {"title":"***","type":1,"url":"1","parentGuid":"toolbar_____"} Could not create - #undefined[***](1) parentId: toolbar_____: Invalid bookmark: {"title":"***","type":1,"url":"1","parentGuid":"toolbar_____"} Could not create - #undefined[wiki-off](1) parentId: toolbar_____: Invalid bookmark: {"title":"wiki-off","type":1,"url":"1","parentGuid":"toolbar_____"} Could not create - #undefined[](1) parentId: toolbar_____: Invalid bookmark: {"title":"","type":1,"url":"1","parentGuid":"toolbar_____"} Could not create - #undefined[](1) parentId: toolbar_____: Invalid bookmark: {"title":"","type":1,"url":"1","parentGuid":"toolbar_____"} Could not create - #undefined[](1) parentId: toolbar_____: Invalid bookmark: {"title":"","type":1,"url":"1","parentGuid":"toolbar_____"} Could not create - #undefined[](1) parentId: toolbar_____: Invalid bookmark: {"title":"","type":1,"url":"1","parentGuid":"toolbar_____"} Could not create - #undefined[](1) parentId: toolbar_____: Invalid bookmark: {"title":"","type":1,"url":"1","parentGuid":"toolbar_____"} Could not create - #undefined[](1) parentId: toolbar_____: Invalid bookmark: {"title":"","type":1,"url":"1","parentGuid":"toolbar_____"} Could not create - #undefined[](1) parentId: toolbar_____: Invalid bookmark: {"title":"","type":1,"url":"1","parentGuid":"toolbar_____"} Could not create - #undefined[](1) parentId: toolbar_____: Invalid bookmark: {"title":"","type":1,"url":"1","parentGuid":"toolbar_____"} Could not create - #undefined[](1) parentId: toolbar_____: Invalid bookmark: {"title":"","type":1,"url":"1","parentGuid":"toolbar_____"} Could not create - #undefined[](1) parentId: toolbar_____: Invalid bookmark: {"title":"","type":1,"url":"1","parentGuid":"toolbar_____"} Could not create - #undefined[](1) parentId: toolbar_____: Invalid bookmark: {"title":"","type":1,"url":"1","parentGuid":"toolbar_____"} Could not create - #undefined[](1) parentId: toolbar_____: Invalid bookmark: {"title":"","type":1,"url":"1","parentGuid":"toolbar_____"} Could not create - #undefined[](1) parentId: toolbar_____: Invalid bookmark: {"title":"","type":1,"url":"1","parentGuid":"toolbar_____"} Could not create - #undefined[](1) parentId: toolbar_____: Invalid bookmark: {"title":"","type":1,"url":"1","parentGuid":"toolbar_____"} Could not create - #undefined[](1) parentId: toolbar_____: Invalid bookmark: {"title":"","type":1,"url":"1","parentGuid":"toolbar_____"} Could not create - #undefined[](1) parentId: toolbar_____: Invalid bookmark: {"title":"","type":1,"url":"1","parentGuid":"toolbar_____"} Could not create - #undefined[](1) parentId: toolbar_____: Invalid bookmark: {"title":"","type":1,"url":"1","parentGuid":"toolbar_____"} Could not create - #undefined[](1) parentId: toolbar_____: Invalid bookmark: {"title":"","type":1,"url":"1","parentGuid":"toolbar_____"} Could not create - #undefined[](1) parentId: toolbar_____: Invalid bookmark: {"title":"","type":1,"url":"1","parentGuid":"toolbar_____"} Could not create - #undefined[Fax](1) parentId: toolbar_____: Invalid bookmark: {"title":"Fax","type":1,"url":"1","parentGuid":"toolbar_____"} Could not create - #undefined[](1) parentId: toolbar_____: Invalid bookmark: {"title":"","type":1,"url":"1","parentGuid":"toolbar_____"} Could not create - #undefined[](1) parentId: toolbar_____: Invalid bookmark: {"title":"","type":1,"url":"1","parentGuid":"toolbar_____"} Could not create - #undefined[](1) parentId: toolbar_____: Invalid bookmark: {"title":"","type":1,"url":"1","parentGuid":"toolbar_____"} Could not create - #undefined[](1) parentId: toolbar_____: Invalid bookmark: {"title":"","type":1,"url":"1","parentGuid":"toolbar_____"} Could not create - #undefined[](1) parentId: toolbar_____: Invalid bookmark: {"title":"","type":1,"url":"1","parentGuid":"toolbar_____"} Could not create - #9VlUUNVRcH-_[-----](1) parentId: toolbar_____: Invalid bookmark: {"title":"-----","type":1,"url":"1","parentGuid":"toolbar_____"} Could not create - #jtEiQG8wvZQv[-----](1) parentId: toolbar_____: Invalid bookmark: {"title":"-----","type":1,"url":"1","parentGuid":"toolbar_____"} Could not create - #_eggDYX87r0u[-----](1) parentId: toolbar_____: Invalid bookmark: {"title":"-----","type":1,"url":"1","parentGuid":"toolbar_____"} Could not create - #eWz74IcNkrzu[-----](1) parentId: toolbar_____: Invalid bookmark: {"title":"-----","type":1,"url":"1","parentGuid":"toolbar_____"} Could not create - #undefined[******](1) parentId: mobile______: Invalid bookmark: {"title":"*****","type":1,"url":"1","parentGuid":"mobile______"} Last synchronized: 11 hours ago 


// ...
```

### Expected Behavior

Bookmarks should be synchronizing flawlessly.

### To Reproduce

Happens everytime I start my browser.

### Debug log provided

- [ ] I have provided a debug log file

**Comment by marcelklehr** [comment]
I'm on it. This is likely caused by the bookmarks app update yesterday

**Comment by marcelklehr** [comment]
Should be fixed now with bookmarks v13.1.1

**Comment by marcelklehr** [comment]
It should be available in the next hour
